lib/webvalve/fake_service_wrapper.rb

```diff
@@ -1,18 +1,23 @@
 module WebValve
   class FakeServiceWrapper
     # lazily resolve the app constant to leverage rails class reloading
-    def initialize(app_class_name)
-      @app_class_name = app_class_name
+    def initialize(service_config)
+      @service_config = service_config
     end
 
     def call(env)
+      env['PATH_INFO'] = env['PATH_INFO'].gsub(/^#{path_prefix}/, '')
```

Comment:
At this point in time, we're about to call into the sinatra app with a path that looks like `/prefix/real/path` when what we want to forward along is `/real/path`.

This is a byproduct of the way webmock doesn't strip out the base part that you've told it to stub, so we have to look back at the full service_url to figure out what that would've been.

Reply:
(Like, this is coming from whatever `to_rack` does to map the stubs)

Reply:
You want \A instead of ^ right?

Reply:
bump on this, @smudge

Reply:
Oh I just knew I'd get called out on that and did it anyway because I didn't think URL pats supported newliens. 😜 

Yup, I can do `\A`.